compiler/src/iree/compiler/Dialect/HAL/Target/LLVM/LLVMCPUTarget.cpp

```diff
@@ -410,13 +412,38 @@ class LLVMCPUTargetBackend final : public TargetBackend {
         continue;
       }
       func.setDSOLocal(true);
-      func.setLinkage(llvm::GlobalValue::LinkageTypes::InternalLinkage);
```

Comment:
Some CI errors for this

https://github.com/iree-org/iree/actions/runs/3989611477/jobs/6842259299
```
[702/821] Generating check_llvm-cpu_local-task_floor.mlir_module.vmfb from floor.mlir
FAILED: tests/e2e/xla_ops/check_llvm-cpu_local-task_floor.mlir_module.vmfb /work/full-build-dir/tests/e2e/xla_ops/check_llvm-cpu_local-task_floor.mlir_module.vmfb 
cd /work/full-build-dir/tests/e2e/xla_ops && /work/full-build-dir/tools/iree-compile --output-format=vm-bytecode --mlir-print-op-on-diagnostic=false --iree-hal-target-backends=llvm-cpu --iree-input-type=mhlo /work/tests/e2e/xla_ops/floor.mlir -o check_llvm-cpu_local-task_floor.mlir_module.vmfb --iree-hal-executable-object-search-path=\"/work/full-build-dir\" --iree-llvm-embedded-linker-path=\"/work/full-build-dir/third_party/llvm-project/llvm/bin/lld\" --iree-llvm-wasm-linker-path=\"/work/full-build-dir/third_party/llvm-project/llvm/bin/lld\"
lld: error: undefined symbol: floorf
```

Reply:
bah thanks for pointing this out! this stuff is so fiddly x_x